documents/course-delivery.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Help
+subtitle: Teaching Strategies and Course Delivery
+---
+
+### Flipped Classroom
+
+I use a flipped, learner-centered, approach to teach this course, because I 
+think that learning to program has very little to do with me talking and a lot 
+to do with students working on computers. Flipped classes work well for all 
+kinds of content, but I think they work particularly well for computer oriented 
+classes. If you're interested in knowing more take a look at this great
+info-graphic:
+
+[![Flipped
+Classroom](http://knewton.marketing.s3.amazonaws.com/images/infographics/flipped-classroom.jpg "Flipped Classroom")](http://www.knewton.com/flipped-classroom/)
+
+Created by [Knewton](http://www.knewton.com/) and [Column Five
+Media](http://columnfivemedia.com/)
+
+
+### Workshop In-class Format
+
+I've found that some students require a little more motivation of the concepts and strategies than just the `readings/` to complete the `assignments/`, so in some cases I include a brief lecture-style introduction or work through a problem as an example. I disperse my front-of-the-room `material/` into `exercise/`-level chunks for concepts that are sequential in nature.
```

Comment:
Replace with something like: "While this course is designed as a flipped classroom, some motivation and re-exposure to the ideas in the readings is beneficial. Experience suggests that giving a brief introduction to one of the ideas being covered, followed by having the students work a single problem on that topic, is an effective way to accomplish this. Depending on the length of the class session this be repeated 2-3 times to cover that many low level topics. Recommended structure for doing this for some lessons is available in `material/`."